modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -554,9 +627,28 @@ export class Wallets implements IWallets {
     common.validateParams(params, ['walletShareId'], ['overrideEncryptedPrv', 'userPassword', 'newWalletPassphrase']);
 
     let encryptedPrv = params.overrideEncryptedPrv;
-
     const walletShare = (await this.getShare({ walletShareId: params.walletShareId })) as any;
-
+    if (walletShare.keychainOverrideRequired && walletShare.permissions.indexOf('admin') !== -1) {
+      if (_.isUndefined(params.userPassword)) {
+        throw new Error('userPassword param must be provided to decrypt shared key');
+      }
+      const walletKeychain = await this.createKeychain(params.userPassword);
+      const response = await this.updateShare({
+        walletShareId: params.walletShareId,
+        state: 'accepted',
+        keyId: walletKeychain.id,
+      });
+      // If the wallet share was accepted successfully (changed=true), reshare the wallet with the spenders
+      if (response.changed && response.state === 'accepted') {
+        try {
+          await this.reshareOfcAccountWithSpenders(walletShare.wallet, params.userPassword);
```

Comment:
Is `keychainOverrideRequired` only set for a single wallet share on the OFC wallet? Is there a case where two users have this set to true in their wallet share?

Reply:
> Is there a case where two users have this set to true in their wallet share?

Yes, it's possible for two different wallet share requests from different users to have `keychainOverrideRequired` set to true. However, it's important to note that only the Admin APIs, accessible exclusively to the support team, can set this `keychainOverrideRequired` flag.

Reply:
So ideally, it's the responsibility of the support team to ensure that only one invitation is sent at a time.

Reply:
As a follow up, we can come back and put some logic in the admin API (that sends out the invitation) to abort if there is already an active WalletShare for a wallet with the `keychainOverrideRequired` property set to true.

Reply:
> So ideally, it's the responsibility of the support team to ensure that only one invitation is sent at a time

Ideally, we should not rely on this.

Reply:
Please address the concern here. If it's possible to have two admins with the keychain override, then wallet spenders will get invited multiple times when they do not need to. There should be restrictions in place to ensure only one admin at a time is doing this op.

Reply:
@rajucse fyi^

Reply:
@zahin-mohammad 
I had a discussion with the team [here](https://bitgo.slack.com/archives/C06MHA22P08/p1713423281043779) regarding the concern you raised. The team has decided to prioritize picking up the improvement later, and for the time being, we will proceed with the current approach.